topbeat/docs/fields.asciidoc

```diff
@@ -168,12 +167,12 @@ type: int
 
 The amount of CPU time spent in involuntary wait by the virtual CPU while the hypervisor was servicing another processor. Available only on Unix.
 
-[float]
+
 === cpus Fields
 
 This group contains CPU usage per core statistics.
 
-[float]
```

Comment:
@ruflin I agree completely! We need to fix the script or figure out a better way of structuring the Topbeat options. The easiest way to see what's happening is to build the doc without including the float tags, and you'll see that the content is broken into too many topics.  

The float tag is only required in Topbeat because there are more categories of output fields than there are in the other Beats.  You'll notice that we've basically made another hack to the script to avoid ending up with a heading 5 level (which isn't supported in asciidoc). So under "System-Wide Statistics Fields", the "load Fields" section should _logically_ be a heading 4 (====). However, we can't do that because then we would end up with the "load.load1" section being a heading 5, and the doc build won't support that. 

I'm not looking at the script right now, but a possible solution might be to remove the container sections ("load Fields" and "cpu Fields" and modify the intro text under "System-Wide Statistics" to say something like:

Contains system-wide statistics. These statistics are the details that you can get by running the top command on Unix systems. The system-wide statistics include: 
* load fields that show the system load average. The load average is the average number of jobs in the run queue.
* cpu fields that contain statistics related to CPU usage.

Then you could just list all the load fields followed by the cpu fields without headings to separate them. 

Does this make sense? Is it possible to change the script to do this?